tests/RobotFramework/tests/cumulocity/log/log_operation_child.robot

```diff
@@ -12,29 +12,33 @@ Test Tags           theme:c8y    theme:log
 
 *** Test Cases ***
 Successful log operation
-    ${end_timestamp}=    get current date    result_format=%Y-%m-%dT%H:%M:%S+0000
+    ${start_timestamp}=    Get Current Date    UTC    -24 hours    result_format=%Y-%m-%dT%H:%M:%S+0000
+    ${end_timestamp}=    Get Current Date    UTC    +60 seconds    result_format=%Y-%m-%dT%H:%M:%S+0000
     ${operation}=    Cumulocity.Create Operation
     ...    description=Log file request
-    ...    fragments={"c8y_LogfileRequest":{"dateFrom":"1970-01-01T00:00:00+0000","dateTo":"${end_timestamp}","logFile":"example","searchText":"first","maximumLines":10}}
+    ...    fragments={"c8y_LogfileRequest":{"dateFrom":"${start_timestamp}","dateTo":"${end_timestamp}","logFile":"example","searchText":"first","maximumLines":10}}
     ${operation}=    Operation Should Be SUCCESSFUL    ${operation}    timeout=120
 
 
 *** Keywords ***
 Setup Child Device
     ThinEdgeIO.Set Device Context    ${CHILD_SN}
     Execute Command    sudo dpkg -i packages/tedge_*.deb
-    Execute Command    sudo dpkg -i packages/tedge-log-plugin*.deb
 
     Execute Command    sudo tedge config set mqtt.client.host ${PARENT_IP}
     Execute Command    sudo tedge config set mqtt.client.port 1883
+    Execute Command    sudo tedge config set mqtt.topic_root te
```

Comment:
That needs to be set explicitly? I thought that was the default.

Reply:
I wanted to be explicit (that is generally my preference)